build.gradle

```diff
@@ -15,8 +15,11 @@ repositories {
 }
 
 dependencies {
+	// https://mvnrepository.com/artifact/commons-fileupload/commons-fileupload
+	implementation group: 'commons-fileupload', name: 'commons-fileupload', version: '1.4'
```

Comment:
1) The library was updated last time in 2018.
2) Dou really need this library? 
https://mvnrepository.com/artifact/commons-fileupload/commons-fileupload

Reply:
I have removed it